UndertaleModTool/Scripts/Resource Unpackers/ExportAllSprites.csx

```diff
@@ -43,9 +45,17 @@ async Task DumpSprites()
 
 void DumpSprite(UndertaleSprite sprite)
 {
+    string outputFolder = texFolder;
+    if (useSubDirectoriesPath)
+        Path.Combine(outputFolder, sprite.Name.Content);

```

Comment:
You change the variable name here (should be `useSubDirectories`) and it should be `outputFolder = Path.Combine(...);`